### PR TITLE
Add tests for logic, timestamps, strings.

### PR DESCRIPTION
**Pull request by JimLarson**

### CODE REVIEW

File: tests/simple/testdata/timestamps.textproto [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    expr: "string(timestamp('2009-02-13T23:31:30Z'))"
+    value: { string_value: "2009-02-13T23:31:30Z" }
+  }
+  test {
+    name: "toType"
+    expr: "type(timestamp('2009-02-13T23:31:30Z'))"
+    value: { type_value: "google.protobuf.Timestamp" }
+  }
+}
+section {
```
Comment (TristonianJones) [inline tests/simple/testdata/timestamps.textproto]: Can you also add a copy of the tests for the exact same inputs, but with a timezone argument specified in the helper accessor?

Comment (JimLarson) [reply tests/simple/testdata/timestamps.textproto]: Done.

File: doc/langdef.md [hunk 2]
```diff
@@ -698,6 +698,26 @@ notation like `_+_` where `_` is a placeholder for an argument.
...
+
+Timezones are expressed in the following grammar:
+```grammar
+TimeZone = "UTC" | LongTZ | FixedTZ ;
+LongTZ = ? list available at http://joda-time.sourceforge.net/timezones.html ? ;
+FixedTZ = ( "+" | "-" ) Digit Digit ":" Digit Digit ;
+Digit = "0" | "1" | ... | "9" ;
+```
+
+Fixed timezones are explicit hour and minute offsets from UTC.  Long timezone
```
Comment (TristonianJones) [inline doc/langdef.md]: Was this set of formats double checked against what the other evaluators support? I'm thinking mostly of C++

Comment (JimLarson) [reply doc/langdef.md]: This was the intersection of what the Java and C++ implementations supported, as could be inferred by reading the code.

**Pull request merged**